### PR TITLE
Update query-execution-plan.md

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

Update the documentation for query-execution-plan.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3201/ , part of [ pingcap/docs-cn#3170](https://github.com/pingcap/docs-cn/pull/3170)
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by ireneontheway** [comment]
PTAL @yikeke

File: query-execution-plan.md [hunk 1]
```diff
@@ -178,32 +184,42 @@ In the above example, the child node of the `TableReader_7` operator is `Selecti
...
+
+In the IndexMerge access mode, the optimizer can use multiple indexes in a table, and combine the returned results of each index to generate the execution plan of the latter IndexMerge in the figure above. Here the `IndexMerge_16` operator has three child nodes, among which `IndexRangeScan_13` and `IndexRangeScan_14` get all the `RowID`s that meet the conditions based on the result of range scan, and then the `TableRowIDScan_15` operator accurately reads all the data that meet the conditions according to these `RowID`s.
+
+For the table scan that is performed by range such as indexRangeScan/TableRangeScan , the operator info column in the explain table has more information about the range of the scanned data than other scan operations. In the above example, the `range:(1,+inf]` in the IndexRangeScan operator indicates that the operator scans the data from 1 to positive infinity.
```
Comment (ran-huang) [inline query-execution-plan.md:214]: `其中对于 IndexRangeScan/TableRangeScan 一类按范围进行的扫表操作来说，explain 表中 operator info 列相比于其他扫表操作，多了被扫描数据的范围这一信息` 
@zz-jason PTAL. [这一句原文](https://github.com/pingcap/docs-cn/pull/3201/files#diff-7958dda55af6cba3f29e9e06ddd9d583R179)表达不太清晰，是指把 `operator info 列`和`其他扫表操作`进行比较吗？还是把 `IndexRangeScan/TableRangeScan` 和`其他扫表操作`进行比较呢？

File: query-execution-plan.md [hunk 2]
```diff
@@ -239,6 +255,8 @@ Generally speaking, `Hash Aggregate` is executed in two stages.
 - One is on the Coprocessor of TiKV/TiFlash, with the intermediate results of the aggregation function calculated when the table scan operator reads the data.
 - The other is at the TiDB layer, with the final result calculated through aggregating the intermediate results of all Coprocessor Tasks.
 
+The operator info column in the explain table also records other information about Hash Aggregation. You need to pay attention to what aggregation function that Aggregation uses. In the above example, the operator info of the Hash Aggregation operator is `funcs:count(Column#7)->Column#4`. It means that Hash Aggregation uses the aggregation function `count` for calculation. The operator info of the Stream Aggregation operator in the following example is the same with this one.
```
Comment (ran-huang) [inline query-execution-plan.md:258]:
```suggestion
The operator info column in the `explain` table also records other information about `Hash Aggregation`. You need to pay attention to what aggregate function that `Hash Aggregation` uses. In the above example, the operator info of the `Hash Aggregation` operator is `funcs:count(Column#7)->Column#4`. It means that `Hash Aggregation` uses the aggregate function `count` for calculation. The operator info of the `Stream Aggregation` operator in the following example is the same with this one.
```

File: query-execution-plan.md [hunk 3]
```diff
@@ -309,6 +327,8 @@ The execution process of `Hash Join` is as follows:
 4. Use the data of the `Probe` side to probe the Hash Table.
 5. Return qualified data to the user.
 
+The operator info column in the explain table also records other information about Hash Join, including whether the query is Inner Join or Outer Join, and what are the conditions of join. In the above example, the query is an Inner Join, where the Join condition `equal:[eq(test.t1.id, test.t2.id)]` partly corresponds with the query statement `where t1.id = t2. id`.The operator info of the other Join operators in the following example is similar to this one.
```
Comment (ran-huang) [inline query-execution-plan.md:330]:
```suggestion
The operator info column in the `explain` table also records other information about `Hash Join`, including whether the query is Inner Join or Outer Join, and what are the conditions of Join. In the above example, the query is an Inner Join, where the Join condition `equal:[eq(test.t1.id, test.t2.id)]` partly corresponds with the query statement `where t1.id = t2. id`. The operator info of the other Join operators in the following examples is similar to this one.
```

File: query-execution-plan.md [hunk 4]
```diff
@@ -470,9 +490,14 @@ EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 00:00:00
 
 After adding the index, use `IndexScan_24` to directly read the data that meets the `start_date BETWEEN '2017-07-01 00:00:00' AND '2017-07-01 23:59:59'` condition. The estimated number of rows to be scanned decreases from 19117643.00 to 8166.73. In the test environment, the execution time of this query decreases from 50.41 seconds to 0.01 seconds.
 
+## Operator-related system variables
+
+Based on MySQL, TiDB defines some special system variables and syntax to optimize performance. Some system variables are related to specific operators, such as the concurrency of the operator, the upper limit of the operator memory, and whether to use partition tables. These can be controlled by system variables, thereby affecting the efficiency of each operator.
```
Comment (ran-huang) [inline query-execution-plan.md:495]:
```suggestion
Based on MySQL, TiDB defines some special system variables and syntax to optimize performance. Some system variables are related to specific operators, such as the concurrency of the operator, the upper limit of the operator memory, and whether to use partitioned tables. These can be controlled by system variables, thereby affecting the efficiency of each operator.
```
Both `partition table` and `partitioned table` are okay, but we prefer the latter in PingCAP docs~

**Comment by ran-huang** [comment]
I just found that most of the changes can be copied from `index-merge.md` 🤔
@ireneontheway Could you please check [this file](https://github.com/pingcap/docs/blob/master/index-merge.md) and align some expressions? I think we'd better make use of the existing translation.

**Comment by ireneontheway** [comment]
PTAL @ran-huang

File: query-execution-plan.md (hunk 1 above)
Comment (zz-jason) [reply query-execution-plan.md:214]: 是 xxxRangeScan 和 其他扫表（Scan）操作进行比较。现在扫表操作总共有这些：

- IndexRangeScan
- IndexFullScan
- TableRangeScan
- TableFullScan
- TableRowIDScan

File: query-execution-plan.md [hunk 5]
```diff
@@ -178,32 +184,44 @@ In the above example, the child node of the `TableReader_7` operator is `Selecti
...
+
+`IndexMerge` allows the optimizer to use multiple indexes per table, and merge the results returned by each index to generate the execution plan of the latter `IndexMerge` in the figure above. Here the `IndexMerge_16` operator has three child nodes, among which `IndexRangeScan_13` and `IndexRangeScan_14` get all the `RowID`s that meet the conditions based on the result of range scan, and then the `TableRowIDScan_15` operator accurately reads all the data that meets the conditions according to these `RowID`s.
+
+For the table scan that is performed by range such as indexRangeScan/TableRangeScan , the operator info column in the explain table has more information about the range of the scanned data than other scan operations. In the above example, the `range:(1,+inf]` in the IndexRangeScan operator indicates that the operator scans the data from 1 to positive infinity.
```
Comment (zz-jason) [inline query-execution-plan.md:214]:
```suggestion
For the scan operation that is performed on a specific range of data, such as `IndexRangeScan`/`TableRangeScan`, the `operator info` column in the result has additional information about the scan range compared with other scan operations like `IndexFullScan`/`TableFullScan`. In the above example, the `range:(1,+inf]` in the `IndexRangeScan_13` operator indicates that the operator scans the data from 1 to positive infinity.
```

File: query-execution-plan.md (hunk 1 above)
Comment (ran-huang) [reply query-execution-plan.md:214]: @ireneontheway 请对应修改英文，谢谢～

File: query-execution-plan.md (hunk 5 above)
Comment (ireneontheway) [reply query-execution-plan.md]: @ran-huang PTAL

**Comment by ran-huang** [comment]
@ireneontheway To align with https://github.com/pingcap/docs-cn/pull/3201, please also delete `index-merge.md` and remove the link from `TOC.md`

File: index-merge.md [hunk 6]
```diff
@@ -1,100 +0,0 @@
----
-title: Access Tables Using `IndexMerge`
-summary: Learn how to access tables using the `IndexMerge` query execution plan.
-aliases: ['/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/']
```
Comment (yikeke) [inline index-merge.md:4]: Please add the following aliases into https://github.com/pingcap/docs/pull/3359/files#diff-7958dda55af6cba3f29e9e06ddd9d583R4:

'/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/','/tidb/dev/index-merge' @ireneontheway

Comment (yikeke) [reply index-merge.md:4]: Please note: When we delete a file, in most cases, we need to add an alias to another document to redirect the old URLs. @ireneontheway @ran-huang

Comment (yikeke) [reply index-merge.md:4]: To fix https://github.com/pingcap/docs-cn/pull/3201/, please help add the missing aliases to query-execution-plan.md in the docs-cn repo. @ireneontheway

File: query-execution-plan.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
 ---
 title: Understand the Query Execution Plan
 summary: Learn about the execution plan information returned by the `EXPLAIN` statement in TiDB.
-aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/understanding-the-query-execution-plan/']
+aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/understanding-the-query-execution-plan/','/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/']
```
Comment (yikeke) [inline query-execution-plan.md:4]:
```suggestion
aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/understanding-the-query-execution-plan/','/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/','/tidb/dev/index-merge']
```

File: query-execution-plan.md [hunk 8]
```diff
@@ -178,32 +184,44 @@ In the above example, the child node of the `TableReader_7` operator is `Selecti
 
 #### `IndexMerge` example
 
-{{< copyable "sql" >}}
+`IndexMerge` is a method introduced in TiDB v4.0 to access tables. Using this method, the TiDB optimizer can use multiple indexes per table and merge the results returned by each index. In some scenarios, this method makes the query more efficient by avoiding full table scans.
 
-```sql
-set @@tidb_enable_index_merge = 1;
-explain select * from t use index(idx_a, idx_b) where a > 1 or b > 1;
 ```
```
Comment (yikeke) [inline query-execution-plan.md:189]:
```suggestion
```sql
```

**Pull request merged**